modules/ROOT/pages/schema/constraints/list-constraints.adoc

```diff
@@ -76,28 +76,26 @@ SHOW CONSTRAINTS
-| 16 | "movie_tagline"          | "NODE_PROPERTY_TYPE"               | "NODE"         | ["Movie"]      | ["tagline"]                  | NULL                    | "STRING | LIST<STRING NOT NULL>" |
-| 14 | "movie_title"            | "NODE_PROPERTY_TYPE"               | "NODE"         | ["Movie"]      | ["title"]                    | NULL                    | "STRING"                         |
-| 27 | "node_uniqueness_param"  | "NODE_PROPERTY_UNIQUENESS"         | "NODE"         | ["Book"]       | ["prop1"]                    | "node_uniqueness_param" | NULL                             |
-| 2  | "node_vector_constraint" | "NODE_PROPERTY_TYPE"               | "NODE"         | ["Movie"]      | ["embedding"]                | NULL                    | "VECTOR<INTEGER32 NOT NULL>(42)" |
-| 21 | "ownershipId"            | "RELATIONSHIP_KEY"                 | "RELATIONSHIP" | ["OWNS"]       | ["ownershipId"]              | "ownershipId"           | NULL                             |
-| 15 | "part_of"                | "RELATIONSHIP_PROPERTY_TYPE"       | "RELATIONSHIP" | ["PART_OF"]    | ["order"]                    | NULL                    | "INTEGER"                        |
-| 17 | "part_of_tags"           | "RELATIONSHIP_PROPERTY_TYPE"       | "RELATIONSHIP" | ["PART_OF"]    | ["tags"]                     | NULL                    | "STRING | LIST<STRING NOT NULL>" |
-| 11 | "prequels"               | "RELATIONSHIP_PROPERTY_UNIQUENESS" | "RELATIONSHIP" | ["PREQUEL_OF"] | ["order", "author"]          | "prequels"              | NULL                             |
-| 28 | "rel_exist_param"        | "RELATIONSHIP_PROPERTY_EXISTENCE"  | "RELATIONSHIP" | ["WROTE"]      | ["published"]                | NULL                    | NULL                             |
-| 3  | "rel_vector_constraint"  | "RELATIONSHIP_PROPERTY_TYPE"       | "RELATIONSHIP" | ["CONTAINS"]   | ["embedding"]                | NULL                    | "VECTOR<FLOAT32 NOT NULL>(1536)" |
-| 7  | "sequels"                | "RELATIONSHIP_PROPERTY_UNIQUENESS" | "RELATIONSHIP" | ["SEQUEL_OF"]  | ["order"]                    | "sequels"               | NULL                             |
-| 13 | "wrote_year"             | "RELATIONSHIP_PROPERTY_EXISTENCE"  | "RELATIONSHIP" | ["WROTE"]      | ["year"]                     | NULL                    | NULL                             |
-+--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
++-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| id | name                    | type                               | entityType     | labelsOrTypes  | properties                   | enforcedLabel | ownedIndex              | propertyType                     |
```

Comment:
for all three of these tables, shouldn't they also include the vector constraints? We have a sentence at the top of the file that says
```
The constraints listed on this page are created on the page xref:schema/constraints/create-constraints.adoc[].
```
so I would think the new vector ones should be listed as well as the graph type related updates 🤔 

 (might need to be added to the setup at the start of the file to actually exist, `The below constraints are created on schema/constraints/create-constraints.adoc`)